docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -2,33 +2,33 @@
 
 = Performing a fast-forward upgrade on Compute services
 
-Compute services rolling upgrade cannot be done during adoption,
-there is in a lock-step with Compute control plane services, because those are managed independently by data plane ansible and Kubernetes Operators.
-The Compute service operator and OpenStack Operator ensure upgrading
-is done independently of each other, by configuring
-`[upgrade_levels]compute=auto` for Compute services. Compute control plane
-services apply the change right after custom resource (CR) is patched. Compute data plane services will catch up the same config change with ansible deployment later on.
+You must upgrade the Compute services from {rhos_prev_long} {rhos_prev_ver} to {rhos_long} {rhos_curr_ver} on the control plane and data plane. The upgrade procedure involves the following tasks:
```

Comment:
You must upgrade the Compute services from {rhos_prev_long} {rhos_prev_ver} to {rhos_long} {rhos_curr_ver} on the control plane and data plane, by completing the following tasks: